climate-monitoring-client/src/main/java/climatemonitoring/EditProfile.java

```diff
@@ -1,18 +1,125 @@
 package climatemonitoring;
 
+import climatemonitoring.core.ConnectionLostException;
+import climatemonitoring.core.DatabaseRequestException;
+import climatemonitoring.core.Operator;
 import climatemonitoring.core.ViewState;
+import climatemonitoring.core.headless.Console;
 
 class EditProfile extends ViewState {
 
 	@Override
 	public void onHeadlessRender(String args) {
 
-		throw new UnsupportedOperationException("Unimplemented method 'onHeadlessRender'");
+		Console.write("User ID > " + m_userID);
+		Console.write("SSID > " + m_ssid);
+		Console.write("Surname > " + m_surname);
+		Console.write("Name > " + m_name);
+		Console.write("Email > " + m_email);
+		Console.write("Password > " + m_password);
+		Console.write("Center ID > " + m_centerID);
+
+		String errorMsg = null;
+
+		do {
+
+			String newSSID = Console.read("New SSID > ");
+
+			if (newSSID != m_ssid) {
+
+				errorMsg = Check.ssidUnique(m_ssid);
+				m_ssid = newSSID;
```

Comment:
Just use the `Check.ssid()` method, because in a few days it will be the same as `Check.ssidUnique()` + this last one will be removed

Reply:
Fixed